input/docs/getting-started/running-cake-scripts.md

```diff
@@ -0,0 +1,45 @@
+Order: 10
+---
+
+There are different runners available for running Cake scripts.
+
+# .NET Core Tool
+
+:::{.alert .alert-info}
+This is the recommended way to run Cake Scripts.
+:::
+
+The [Cake.Tool](https://www.nuget.org/packages/Cake.Tool) NuGet package, is a .NET Core global tool compiled for .NET Core 2.1 or newer.
+
+It allows you to easily install Cake and have it globally available in your environment using the dotnet tool command:
+
+```shell
+dotnet tool install -g Cake.Tool --version x.y.z
+``` 
+
+Once installed, you can launch Cake from anywhere using the .NET CLI:
+
+```shell
+dotnet cake build.cake
```

Comment:
Should we remove `build.cake` here and perhaps just mention that it will run `build.cake` by convention?

Reply:
Updated the example and added a note mentioning that it calls `build.cake` by convention and that it can be overwritten.